src/rbac/defaultRoleManager.ts

```diff
@@ -70,7 +70,7 @@ export class DefaultRoleManager implements RoleManager {
     }
 
     if (!this.hasRole(name1) || !this.hasRole(name2)) {
-      throw new Error('error: name1 or name2 does not exist');
+      return;
```

Comment:
Why remove the error?

Reply:
I think it is not an error,  so remove this error.